content/non-web-ict.md

```diff
@@ -30,6 +30,8 @@ This page introduces the document: "Guidance on Applying WCAG 2.0 to Non-Web Inf
 
 Quick link: [WCAG2ICT Note](https://www.w3.org/TR/wcag2ict/)
```

Comment:
I think that this Quick Link should be specifically linking to the currently published WCAG2ICT Note (dated Sept. 2015) with text that indicates the version date.